src/relax/transform/static_plan_block_memory.cc

```diff
@@ -476,11 +521,15 @@ class StorageAllocatorInit : public StorageAllocatorBaseVisitor {
     token2block_.erase(token_to_discard.get());
   }
 
+  /*! \brief The arithmetic analyzer. */
+  arith::Analyzer ana;
```

Comment:
ana_

Reply:
Ah thanks my bad. It was added in the last minute so missed this point.